### DISCUSSION TITLE
Our university ID does not comply with GitHub verification process.

### DISCUSSION THREAD

**Discussion by KRVS-Engineering** [community/community#55558] [GitHub Education]
I work for the University of Louisiana at Lafayette and our ID does not have the name of the university listed on the ID.  It just has a photo of the main university building on the front.  This will not pass the automated verification process.  Please advise if there is another method I can use.  I can submit my ID (front and back) again if needed.

Thank in advance,

kw

**Comment by samus-aran** [comment]
Hi @KRVS-Engineering 

You can take an image of multiple pieces of evidence. So if you have a Student ID but require a school name then include a supporting piece of evidence with the name on it along side the Student ID.

Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance 👋

**Comment by KRVS-Engineering** [comment]
I cannot seem to find the education certification panel again.  I see the upgrade path to pro but nothing about educational.  Attached is a copy of my ID, Business card and our license showing the University of Louisiana at Lafayette as our license holder with the FCC.  We always find ourselves in a tough spot because we are not a 501c3 nor are we 100% academic although this project is.

Thank you for your assistance and sorry for the trouble.

Kw
